constellation/src/storage/rocks_store.rs

```diff
@@ -35,16 +35,38 @@ fn rocks_opts_base() -> Options {
     let mut opts = Options::default();
     opts.set_level_compaction_dynamic_level_bytes(true);
     opts.create_if_missing(true);
+
+    let mut bopts = rocksdb::BlockBasedOptions::default();
+    bopts.set_cache_index_and_filter_blocks(true);
+    bopts.set_pin_l0_filter_and_index_blocks_in_cache(true);
+    bopts.set_bloom_filter(10.0, false);
+    bopts.set_checksum_type(rocksdb::ChecksumType::XXH3);
```

Comment:
I _think_ the default is CRC32; XXH3 is much faster. Considering constellation is probably IO-bound (on the RasPi), most block reads come from the OS page cache or disk directly, so each block read will incur a checksum calculation.